File: fizzbuzz.py
# FizzBuzz Solution for Python
#  - submitted by @jimeh

for i in xrange(1, 100):
	o = ""
	if i % 3 == 0: o += "Fizz"
	if i % 5 == 0: o += "Buzz"
	print(i if o == "" else o)

File: fizzbuzz.rb
# FizzBuzz Solution for Ruby
#  - submitted by @jimeh

(1..100).each do |i|
	o = ""
	o << "Fizz" if i % 3 == 0
	o << "Buzz" if i % 5 == 0
	puts o == "" ? i : o
end